Core/Utilities/Graphs/GraphData.py

```diff
@@ -316,65 +318,65 @@ def truncateLabels( self, limit = 10 ):
         for key in self.all_keys:
           if key in self.subplots[label].parsed_data:
             other_data[key] += self.subplots[label].parsed_data[key]
-    self.otherPlot = PlotData( other_data )
+    self.otherPlot = PlotData(other_data)
 
-  def getStats( self ):
+  def getStats(self):
     """ Get statistics of the graph data
     """
 
-    numData = self.getPlotNumData( zipFlag = False )
-    if not numData:
+    numData = self.getPlotNumData(zipFlag=False)
+    if not len(numData):
```

Comment:
As pylint usually complains about this check, maybe add a comment, and the relevant `  # pylint: disable=len-as-condition`?